docs/dev-notes/DEVELOPMENT.md

```diff
@@ -0,0 +1,44 @@
+# Development
+
+### Getting started
+
+Since this SDK is npm package, it does not need to be compiled separately.
+
+
+## Work on Ami App locally
+
+We use the [Ami App](https://github.com/customerio/amiapp-reactnative) to test the SDK in a real-world environment.
+
+When you install dependencies via yarn/npm in react native app, you can install it from your local machine directly. You need to follow the following steps to use local version of the SDK
+
+- Go to `package.json` of your react native app
+- Find the SDK dependency i.e. `"customerio-reactnative": "{version}"`
+- Update the SDK version to path of the SDK e.g. `"customerio-reactnative": "../customerio-reactnative"`
```

Comment:
Instead of doing this, what about using `npm link` command? 

I *think* when you use `npm link`, you don't need to modify your `package.json` file at all. Which would be nice to not accidentally commit changes to `package.json` file you didn't mean to.

Reply:
This was my next action item to explore more options on it, thanks for sharing. I'll take a look and try to make local testing easier so we don't accidentally commit any unwanted changes.

Reply:
`npm link` doesn't seem to be working because of symlinks. I need time to improve this and have [created an issue](https://github.com/customerio/issues/issues/9350) to track and improve this in a separate PR. If you have more ideas, please keep posting them on the issue. I'll pick this up whenever I get some time to try out the suggested ideas.